src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -274,8 +274,9 @@ void func_8096FAAC(EnKakasi* this, PlayState* play) {
  * goes off every frame of song teach, but... doing what?
  */
 void func_8096FBB8(EnKakasi* this, PlayState* play) {
-    if ((play->msgCtx.unk12048 == 0) || (play->msgCtx.unk12048 == 1) || (play->msgCtx.unk12048 == 2) ||
-        (play->msgCtx.unk12048 == 3) || (play->msgCtx.unk12048 == 4)) {
+    if ((play->msgCtx.lastOcarinaButtonIndex == 0) || (play->msgCtx.lastOcarinaButtonIndex == 1) ||
+        (play->msgCtx.lastOcarinaButtonIndex == 2) || (play->msgCtx.lastOcarinaButtonIndex == 3) ||
```

Comment:
```suggestion
    if ((play->msgCtx.lastOcarinaButtonIndex == OCARINA_BTN_A) || (play->msgCtx.lastOcarinaButtonIndex == OCARINA_BTN_C_DOWN) ||
        (play->msgCtx.lastOcarinaButtonIndex == 2) || (play->msgCtx.lastOcarinaButtonIndex == 3) ||
```
Could you fill out the enum for this now that it's being named?